docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.2"
+services:
+
+  redis:
+    image: redis:6-alpine
+
+  web:
+    image: rt-playground
+    build: .
+    depends_on:
+      - redis
+    volumes:
+      - .:/usr/src/app
+    env_file:
+      - .env
+    environment:
+      REDIS_URL: redis://redis:6379
+      HOST: 0.0.0.0
```

Comment:
istn this all in .env now?

Reply:
it's different there - i.e. redis is localhost, while in docker is on `redis` host, and host is localhost while for docker needs to be on all interfaces.

Reply:
If you expose the port, the server can be still be started on localhost, i guess?